examples/2d/elixir_advection_timeintegration.jl

```diff
@@ -67,5 +67,5 @@ callbacks = CallbackSet(summary_callback,
 ode_algorithm = Trixi.CarpenterKennedy2N54()
 sol = Trixi.solve(ode, ode_algorithm,
                   dt=1.0, # solve needs some value here but it will be overwritten by the stepsize_callback
-                  save_everystep=false, callback=callbacks);
+                  save_everystep=false, callback=callbacks, maxiters=typemax(Int));
```

Comment:
The docs for DifferentialEquations.jl [says](https://diffeq.sciml.ai/stable/basics/common_solver_opts/#Miscellaneous) that the default value for `maxiters` is `1e5`. This is thus what we have been using in other elixirs as well. For consistency I propose to do the same here, or change it to `typemax(Int)` in the other elixirs as well.

Reply:
Good point! Done in 38af098